### ISSUE TITLE
High GPU usage when running several Collection Runners

### ISSUE THREAD

**Issue by arlemi** [a85/POSTMan-Chrome-Extension#8198]
Reporting on behalf of a user: https://twitter.com/MarioHachemer/status/1240593611777093633

**Describe the bug**
The Collection Runner consumes a lot of GPU resources.

**To Reproduce**
1. Run 4 collection runners in parallel
2. Check the Task Manager

**Expected behavior**
Collection Runner could be taking less computing resources.

**Screenshots**
![ETd5TtrWoAA91uq](https://user-images.githubusercontent.com/5029719/77077772-9703dd80-69ed-11ea-977b-cf2e818e306b.png)

**App information (please complete the following information):**
To be confirmed
 - App Type: Native App
 - Postman Version 7.20.1
 - OS: Windows 10

**Additional context**

**Comment by Roritharr** [comment]
More Details on the Machine and Postman Version:
![image](https://user-images.githubusercontent.com/561299/77083876-75f7b880-69fe-11ea-9687-b47436545573.png)

**Comment by MC-sanderv** [comment]
Closing the issue due to being stale.
Performance issues for Postman have been reported since this issue was opened also, so I will close this one. 
There have been many changes and upgrades to Postman since this was reported. 

Please open a new ticket with relevant information should this also be a problem in the latest Postman versions.
